x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/state.ts

```diff
@@ -9,21 +9,32 @@ import { checkPermission } from '../../../../../privilege/check_privilege';
 
 import { DataFrameAnalyticsId, DataFrameAnalyticsConfig } from '../../../../common';
 
-const ANALYTICS_DETAULT_MODEL_MEMORY_LIMIT = '50mb';
+const OUTLIER_DETECTION_DEFAULT_MODEL_MEMORY_LIMIT = '50mb';
+const REGRESSION_DEFAULT_MODEL_MEMORY_LIMIT = '100mb';
 
 export type EsIndexName = string;
+export type DependentVariable = string;
 export type IndexPatternTitle = string;
+export type AnalyticsJobType = JOB_TYPES | undefined;
 
 export interface FormMessage {
   error?: string;
   message: string;
 }
 
+export enum JOB_TYPES {
```

Comment:
This is somewhat redundant with `ANALYSIS_CONFIG_TYPE` from `x-pack/legacy/plugins/ml/public/data_frame_analytics/common/analytics.ts` can you reuse that one?

Reply:
The other one contains the `UNKNOWN` type - which we don't want since this enum is used to populate the job type options for the select. That's why I've kept them separate.

Reply:
oh I see! Nit: I think it would make sense to have them in one place/file though (just thinking of more possible future job types where we might have to hunt this down in different files)

Reply:
Yes, good call - happy to move that in in the follow-up. Will keep an eye out for any other common types else that could be moved to an easier to find central place